src/exec.spec.js

```diff
@@ -52,20 +53,58 @@ describe('exec', () => {
     //  Fake the win32 platform.
     Object.defineProperty(process, 'platform', { value: 'win32' });
 
+    // //  We expect 'childProcess.exec' to be called with 'which'.
+    // sandbox.stub(childProcess, 'exec')
```

Comment:
A hand here please!
If I uncomment this block this test fails and makes the next one fail too.